calyx/src/passes/inliner.rs

```diff
@@ -1,13 +1,14 @@
-use crate::lang::component::Component;
-use crate::lang::{
-    ast, context::Context, structure, structure_builder, structure_iter,
+use crate::{
+    errors::Error,
+    frontend::library::ast::LibrarySignatures,
+    ir::{
+        analysis::Analysis,
+        traversal::{Action, Named, VisResult, Visitor},
+        Assignment, Builder, Component, Control, Guard, Port,
```

Comment:
Prefer importing `crate::ir` and using `ir::Assignment` etc. instead of direct names.